### PR TITLE
Transparent image and removed dubble image

### PR DESCRIPTION
**Pull request by jeroentjuuhh**
Two small changes:
- Made the background image of the intro transparent.
- Removed a image that was there twice

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -98,12 +100,25 @@ button:active {
 }
 
 .intro > div {
-  background: url("images-optimized/cheers.png") no-repeat center;
-  background-size: 50%;
+  /* background: url("images-optimized/cheers.png") no-repeat center; */
+  /* background-size: 50%; */
```
Comment (TimvdLippe) [inline style.css]: Do not leave commented code

File: style.css [hunk 2]
```diff
@@ -55,6 +55,8 @@ section:nth-child(2n) {
   font-family: "Helvetica Neue", "Calibri Light", Roboto, sans-serif;
 }
 
+
+
```
Comment (TimvdLippe) [inline style.css]: Unnecessary whitespace

File: index.html [hunk 3]
```diff
@@ -24,7 +24,10 @@
   <main>
     <section class="intro" id="">
       <div>
-        <h1 class="cheers">Cheers</h1>
+        <h1 class="cheers">CHeers!</h1>
+        <br>
+        <br>
+        <br>
```
Comment (TimvdLippe) [inline index.html]: Do not use `<br>` for spacing (you should almost never use `<br>`). Instead, set the height in the CSS.

File: style.css [hunk 4]
```diff
@@ -133,14 +148,23 @@ button:active {
...
 @media (max-width: 400px) {
   .leaderboard {
     min-width: 100%;
   }
-}
-.leaderboard ol {
-  font-size: 4vw;
-  margin-left: 50px;
+  .leaderboard ol {
+    font-size: 4vw !important;
```
Comment (TimvdLippe) [inline style.css]: Do not use `!important`. It should not be necessary to add this. Moreover, it negates the selector as specified on line 170, which would break 4k screens.

**Comment by jeroentjuuhh** [comment]
Changed the requested changes.
### Requested changes:
- Removed unnecessary white spaces and commented code
- Removed the !important and created another solution for that.
- Implemented another solution to the <br> solution.

### Also done: 
- Changed the leaderboard names.

Hope these changes are correct!
@TimvdLippe

File: index.html [hunk 5]
```diff
@@ -25,10 +25,7 @@
     <section class="intro" id="">
       <div>
         <h1 class="cheers">CHeers!</h1>
-        <br>
-        <br>
-        <br>
-        <h2> It has begun!</h2>
+        <h2 style="padding-top: 200px"> It has begun!</h2>
```
Comment (TimvdLippe) [inline index.html]: Put this styling in the `styles.css`

**Comment by TimvdLippe** [comment]
@jeroentjuuhh Please check how I reformulated the PR description to improve the eventual commit message in https://github.com/WISVCH/dies-2018/commit/451f18964add56376b9efafcd8cdc857fbe0d59f
